iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -242,6 +263,17 @@ static void createNBBreaker(int node1, int node2, String suffix, String prefix,
                 .add();
     }
 
+    static void createNBDisconnector(int node1, int node2, String id, VoltageLevel.NodeBreakerView view, boolean open) {
+        view.newSwitch()
+                .setId(id)
+                .setEnsureIdUnicity(true)
+                .setKind(SwitchKind.DISCONNECTOR)
+                .setOpen(open)
+                .setNode1(node1)
+                .setNode2(node2)
+                .add();
+    }
+
     static void createNBDisconnector(int node1, int node2, String suffix, String prefix, VoltageLevel.NodeBreakerView view, boolean open) {
```

Comment:
To avoid code duplication, this method should call your new `createNBDisconnector` method.